demos/cache_sidechannel.cc

```diff
@@ -111,3 +113,12 @@ std::pair<bool, char> CacheSideChannel::RecomputeScores(char safe_offset_char) {
   return std::make_pair((scores_[best_val] > 2 * scores_[runner_up_val] + 40),
                         best_val);
 }
+
+std::pair<bool, char> CacheSideChannel::AddHitAndRecomputeScores() {
+  static size_t additional_offset_counter = 0;
+  // Resulting char is between ASCII code 0 and 127.
+  size_t mixed_i = ((additional_offset_counter * 167) + 13) & 0x7F;
```

Comment:
No reason to assume ASCII here. Let's make this 0xff. Also less likely to collide with our actual ASCII use case!

Reply:
When it's over 127, the cast to size_t gives unpredictable results:
https://stackoverflow.com/questions/25850244/casting-from-size-t-to-char-and-around
How would you overcome that?